python/hail/dataset.py

```diff
@@ -3064,14 +3065,18 @@ def linreg3(self, ys, covariates=[], root='va.linreg', use_dosages=False, varian
                       delta=nullable(numeric),
                       sparsity_threshold=numeric,
                       use_dosages=bool,
-                      n_eigs=nullable(integral),
-                      dropped_variance_fraction=(nullable(float)))
+                      n_eigs=nullable(integral))
     def lmmreg(self, kinshipMatrix, y, covariates=[], global_root="global.lmmreg", va_root="va.lmmreg",
-               run_assoc=True, use_ml=False, delta=None, sparsity_threshold=1.0, use_dosages=False,
-               n_eigs=None, dropped_variance_fraction=None):
```

Comment:
@jbloom22 We can't technically get rid of dropped_variance_fraction the way you did here without braking backwards compatibility in 0.1. I doubt people are using it right now, but it's in 0.1